chatlearn/models/vllm_module.py

```diff
@@ -131,7 +131,7 @@ def _init_args(self):
             max_num_batched_tokens=self.model_args.get("max_num_batched_tokens"),
             max_num_seqs=self.model_args.get("micro_batch_size"),
             max_model_len=self.model_args.get("seq_length"),
-            enforce_eager=True,
+            enforce_eager=self.model_args.get("enforce_eager", False),
```

Comment:
the default value is inconsistent between *.py and *.yaml?

Reply:
enforce_eager=False in .py is used to test for qwen, enforce_eager=True for llama.

Reply:
Configs for different models should be set in *.yaml. `enforce_eager` in *.py should be set as True in my opinion.